### DISCUSSION TITLE
Seeking to Connect with Developers #DataScience

### DISCUSSION THREAD

**Discussion by BCMarquez** [mastodon/mastodon#23083] [General]
Hello Everyone,

Is there a place where Mastodon contributing developers get together to discuss needed or wanted features for Mastodon?
I'm **curious if there are any data science related features or services that are needed/wanted by the Mastodon community** and would like to see if there's anywhere I can help. 
If anyone can point me in the right direction, it'd be much appreciated.

Thanks!

**Comment by PhilipMay** [comment]
Good point. I am also coming from the DS and ML domain and asked myself the same question.

Maybe such a service would not be bound to Mastodon but to all of the ActivityPub network.
Maybe something like an "account recommendation system".

Based on the ppl. you are connected to and what you (and they) are posting it suggests accounts you might be interested in.
This could not just be Mastodon accounts but also Friendica and so on.

**Reply by BCMarquez** [reply]
Recommendation systems is def something that can be pitched. Just think it'd take some thinking considering privacy is a big priority in the Mastodon community. But that's what leads me to my original post.

I was searching for a place to discuss, but I heard from an account called @feditips on the mstd.social server, that Github is where people mainly chat. Maybe I'll change the title of this post or start a new one completely where we can throw ideas out there and gather ppl involved in data sci.
